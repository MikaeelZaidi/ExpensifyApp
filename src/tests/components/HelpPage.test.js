import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import HelpPage from '../../components/ExpenseDashBoardPage'

test('This should test the help page',()=>{
    const wrapper = shallow(<HelpPage/>)
    expect(toJSON(wrapper)).toMatchSnapshot()
})