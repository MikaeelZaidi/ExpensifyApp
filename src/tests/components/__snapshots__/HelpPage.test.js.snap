// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`This should test the help page 1`] = `
<div>
  <Connect(ExpenseListFilter) />
  <ExpenseSummary />
  <Connect(ExpenseList) />
</div>
`;
